src/bee.ts

```diff
@@ -462,4 +464,22 @@ export class Bee {
       upload: uploadSingleOwnerChunkData.bind(null, this.url, canonicalSigner),
     }
   }
+
+  /**
+   * Ping the base bee URL. If connection was not successful throw error
+   */
+  health(): Promise<void> | never {
+    return health.health(this.url)
+  }
+
+  /**
+   * Periodically ping the base bee URL
+   * emit 'check' event each time the ping happens
```

Comment:
```suggestion
   * @emits 'check' event each time the ping happens
```

There is `@emits/@fires` js doc directive ;-)